api/services/rapid_pro_services/update_user_group.py

```diff
@@ -0,0 +1,35 @@
+from api import app, db
+import requests
+from api.models.user_group import UserGroup
+from flask import jsonify
+
+
+class UpdateUserGroup:
```

Comment:
Also update file name
```suggestion
class UpdateUserGroupService:
```